editoast/src/views/rolling_stocks.rs

```diff
@@ -54,7 +64,7 @@ async fn get_livery(db_pool: Data<DbPool>, path: Path<(i64, i64)>) -> Result<Htt
 struct RollingStockCreateForm {
     pub name: String,
     pub version: String,
-    pub effort_curves: JsonValue,
+    pub effort_curves: DieselJson<EffortCurves>,
```

Comment:
We don't need the `DieselJson` wrapper here.

```suggestion
    pub effort_curves: EffortCurves,
```

We need to adapt the `From` implementation to wrap it there.

Reply:
Done !